packages/openactive-integration-tests/test/helpers/flow-helper.js

```diff
@@ -13,52 +17,86 @@ const pMemoize = require("p-memoize");
 class FlowHelper {
   /**
    * @param {InstanceType<import('./request-state')['RequestState']>} helper
+   * @param {object} [options]
+   * @param {Set<StageIdentifier>} [options.stagesToSkip] Any stages included
+   *   in this set will be skipped. e.g. with stagesToSkip = new Set(['c2']),
+   *   the C2 stage will be skipped.
+   *
+   *   This can be used to test unconventional flows
    */
-  constructor (helper) {
+  constructor (helper, { stagesToSkip = new Set() } = {}) {
     this.state = helper;
+    this._stagesToSkip = stagesToSkip
```

Comment:
Something like this...

```suggestion
  constructor (helper, { stagesToSkip = [] } = {}) {
    this.state = helper;
    this._stagesToSkip = new Set(stagesToSkip);
```